### PR TITLE
Feat/new languages

### PR DESCRIPTION
**Pull request by ccactus**
Added options for powershell & python and made sure that the javascript usecase matched them.

### CODE REVIEW

**Comment by chappers00** [comment]
Sorry @ccactus I won't have a chance to review this

File: powershell/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Grad Tech Test - PowerShell
+
+## Clone this repository
+
+Use `git clone` to clone the repository locally, then change the remote, please do not fork this repository. To change the remote, use
+
+```bash
```
Comment (Xorima) [inline powershell/README.md:7]:
```suggestion
```powershell
```

File: powershell/medals.ps1 [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        [array]
+        $Medals
+    )
+
+    # Add your code here
+    # Create a medal table based on the provided result list
+    # First place gives 3 points, second place 2 points and third place 1 point
+}
+
+Write-Host "Done."
```
Comment (Xorima) [inline powershell/medals.ps1:15]: Why do we have the `write-host` lines here?

Comment (ccactus) [reply powershell/medals.ps1:15]: Pattern copied from an example elsewhere, happy to drop them!

**Review by Xorima — APPROVED** [review]

**Pull request merged**